docs/en/fuse-sdk/Local-Cache-Quick-Start.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: global
+title: FUSE SDK with Local Cache Quick Start
+group: FUSE SDK
+priority: 2
+---
+
+* Table of Contents
+  {:toc}
+
+## Prerequisites
+
+The followings are the basic requirements running ALLUXIO POSIX API.
+
+- On one of the following supported operating systems
+    * MacOS 10.10 or later
+    * CentOS - 6.8 or 7
+    * RHEL - 7.x
+    * Ubuntu - 16.04
+- Install JDK 11, or newer
+    - JDK 8 has been reported to have some bugs that may crash the FUSE applications, see [issue](https://github.com/Alluxio/alluxio/issues/15015) for more details.
+- Install libfuse
+    - On Linux, we support libfuse both version 2 and 3
+        - To use with libfuse2, install [libfuse](https://github.com/libfuse/libfuse) 2.9.3 or newer (2.8.3 has been reported to also work with some warnings). For example on a Redhat, run `yum install fuse`
+        - To use with libfuse3 (Default), install [libfuse](https://github.com/libfuse/libfuse) 3.2.6 or newer (We are currently testing against 3.2.6). For example on a Redhat, run `yum install fuse3`
+        - See [Select which libfuse version to use](#select-libfuse-version) to learn more about the libfuse version used by alluxio
+    - On MacOS, install [osxfuse](https://osxfuse.github.io/) 3.7.1 or newer. For example, run `brew install osxfuse`
+
+## Build and Install
+
+Option 1. Build the whole Alluxio tarball and launch from.
```

Comment:
since it's quick start, i'd remove the build option and only keep the download option. devs can figure out how to build on their own; that's not quick start anymore

Reply:
Removed